config/config.default.yaml

```diff
@@ -471,6 +471,10 @@ sector:
       upper_threshold_ambient_temperature: 10
       rolling_window_ambient_temperature: 72
       relative_annual_temperature_reduction: 0.01
+    ptes:
+      dynamic_capacity: true
+      max_top_temperature: 90 # °C
+      min_bottom_temperature: # °C
```

Comment:
What should the default value be?

Also: no comments in `config.default.yaml`. All in `doc/configtables*` (and later in json-schema #1547).